aries-site/src/data/structures/templates.js

```diff
@@ -598,3 +598,128 @@ export const templates = [
     tags: ['content', 'layout', 'page layouts', 'responsiveness', 'responsive'],
   },
 ];
+
+// these pages do not necessarily have their own routes but instead
+// link to sections of existing pages
+export const contentLayouts = [
+  {
+    name: 'Single Column - Wizard',
+    category: 'Single Column',
+    description: '',
+    preview: {
+      background: 'background-front',
+      image: {
+        src: {
+          light: '/templateImages/1col-wizard.svg',
+          dark: '/templateImages/1col-wizard-invert.svg',
+        },
+        alt: `Coming soon placeholder with lo-fi wireframe of single column 
+        wizard.`,
```

Comment:
Similar suggestion for other entries if you adopt this one.

```suggestion
        alt: `"Coming Soon" text atop low-fidelity wireframe depicting a single column 
        wizard.`,
```

Reply:
Nice, I'll update them all.